backends/ebpf/runtime/kernel.mk

```diff
@@ -70,7 +72,7 @@ $(BPFNAME).bc: %.bc : %.c
 
 # Invoke the llvm on the generated .bc code and produce bpf byte code
 $(BPFNAME).o: %.o : %.bc
-	$(LLC) -march=bpf -mcpu=probe -filetype=obj $< -o $@
+	$(LLC) -march=bpf -mcpu=probe -filetype=obj $< -o $(BPFNAME)_ebpf.o
```

Comment:
I liked the previous pattern better. This is too specific.

Reply:
I renamed the file to avoid overlaps between the ebpf code test.o and the runtime binary object. Since all files of the runtime are now contained in /build it should be fine. Will revert that change.